README.md

```diff
@@ -1,124 +1,35 @@
-# hear2021-eval-kit
+# hear-preprocess
 
-Evaluation kit for HEAR 2021 NeurIPS competition
+Dataset preprocessing code for the HEAR 2021 NeurIPS competition.
+
+Unless you are a HEAR organizer or want to contribute a task,
+you won't need this repo. Use
+[hear-eval-kit](https://github.com/neuralaudio/hear-eval-kit/) to
+evaluate your embedding models on these tasks.
```

Comment:
Are we going to host these datasets somewhere for participants to use?

Reply:
Yes, on google cloud or huggingaface